.kokoro/tests/run_tests.sh

```diff
@@ -55,7 +55,7 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
 
     # Setup required env variables
     export GOOGLE_CLOUD_PROJECT=java-docs-samples-testing
-    export GOOGLE_CLOUD_PROJECT_NUMBER="779844219229" # For Transcoder samples
+    export TRANSCODER_PROJECT_NUMBER="779844219229" # For Transcoder samples
```

Comment:
Should this be hard coded into the script, or should we source it from the secrets file on Cloud Devrel Kokoro Resources?

Reply:
Yes, this PR is just for updating the test durations due to flakiness.

I'm open to sourcing these if that is the best practice. If so, can you point me to an example?

Reply:
It's fine either way.  Different repo's do different things.  You'll note that the line above is also hardcoded.